decidim-core/spec/lib/engine_spec.rb

```diff
@@ -87,5 +87,12 @@ module Decidim::Core
         end
       end
     end
+
+    it "loads engine and app previews" do
+      expect(ActionMailer::Preview.all).to match_array([
+                                                         Decidim::DeviseMailerPreview,
+                                                         Decidim::Initiatives::InitiativesMailerPreview
+                                                       ])
```

Comment:
```suggestion
      expect(ActionMailer::Preview.all).to include(Decidim::DeviseMailerPreview)
```

Let's change it to only checking the relevant class

Reply:
Done 😄